scripts/bump.sh

```diff
@@ -2,14 +2,14 @@
 
 set -eu
 
-CHANNEL=$1
-echo "CHANNEL: $CHANNEL"
+NPM_TAG=$1
+echo "NPM_TAG: $NPM_TAG"
 
 PRISMA_VERSION=$2
 echo "PRISMA_VERSION: $PRISMA_VERSION"
 
 # TODO: remove this if-condition once we move to dev
-if [ "$CHANNEL" = "dev" ]; then
+if [ "$NPM_TAG" = "dev" ]; then
     PRISMA_CHANNEL="alpha"
```

Comment:
I am confused now @divyenduz - what is CHANNEL/NPM_TAG vs PRISMA_CHANNEL?

Reply:
CHANNEL represents the release path we are on. Its value is similar to `NPM_TAG` but it also represents which release path we are on in VSCode CI i.e. `unstable` or `stable`.

I think `RELEASE_CHANNEL` is probably a better name for this variable. 
Keeping its value similar to Prisma CLI's `NPM_TAG` meant fewer conditions in code but we can change this to `unstable`/`stable`, I personally would keep them as is and just change the variable name to something that is more representative.

Reply:
I'm still a little confused about which variable name should be changed now, rename NPM_TAG back to CHANNEL and leave everything as is, or rename CHANNEL to RELEASE_CHANNEL?

Reply:
I think CHANNEL should be renamed to RELEASE_PATH to make it more clear what is meant here.